docs/pages/deploy/build-project.mdx

```diff
@@ -1,28 +1,124 @@
 import { Terminal } from '~/ui/components/Snippet';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+Whether you have built a native app binary using [EAS](/build/setup/) or [locally](/guides/local-app-development/), the next step in your app development journey is to submit your app to the stores. To do so, you need to create a **production build**.
+
+Production builds are submitted to app stores for release to the general public or as part of a store-facilitated testing process such as TestFlight. This guide explains how to create production builds with [EAS](#production-builds-using-eas) and [locally](#production-builds-locally). It is also possible to create production builds for Expo apps with any CI service capable of compiling Android and iOS apps.
+
+## Production builds using EAS
+
+Production builds must be installed through their respective app stores. They cannot be installed directly on your Android Emulator or device, or iOS Simulator or device. The only exception to this is if you explicitly set `"buildType": "apk"` for Android on your build profile. However, it is recommended to use **aab** when submitting to stores, and this is the default configuration.
+
+### `eas.json` configuration
+
+A minimal configuration for building a production build in **eas.json** that is already created when you create your first build:
+
+{/* prettier-ignore */}
+```json eas.json
+{
+  "build": {
+    /* @hide ... */ /* @end */
+    "production": {}
+    /* @hide ... */ /* @end */
+  }
+}
+```
+
+### Create a production build
+
+To create a production build, run the following command for a platform:
+
+<Tabs tabs={['Android', 'iOS']}>
+  <Tab>
+    <Terminal cmd={['$ eas build --platform android']} />
+  </Tab>
+  <Tab>
+    <Terminal cmd={['$ eas build --platform ios']} />
+  </Tab>
+</Tabs>
+
+> You can attach a message to the build by passing `--message` to the build command, for example, `eas build --platform ios --message "Some message"`. The message will appear on the website. It comes in handy when you want to leave a note with the purpose of the build for your team.
+> Alternatively, you can use `--platform all` option to build for Android and iOS at the same time:
+
+<Terminal cmd={['$ eas build --platform all']} />
+
+## Developer account
+
+You will need to have a developer account for the app store you want to submit your app.
+
+<Collapsible summary="Google Play Developer membership is required to distribute to the Google Play Store.">
+
+You can build and sign your app using EAS Build, but you can't upload it to the Google Play Store unless you have a membership, a one-time $25 USD fee.
+
+</Collapsible>
+
+<Collapsible summary="Apple Developer Program membership is required to build for the Apple App Store.">
+
+If you are going to use EAS Build to create release builds for the Apple App Store, you need access to an account with a $99 USD [Apple Developer Program](https://developer.apple.com/programs) membership.
```

Comment:
```suggestion
If you are going to use EAS Build to create release builds for the Apple App Store, you need access to an account with a $99 USD [Apple Developer Program](https://developer.apple.com/programs) membership.
```

- We are using the term "release" builds here but in this doc we don't specify that production builds are aka release builds. Maybe we should do it in the introduction.

Reply:
I changed this to be "production" instead of "release"